interface/index.ts

```diff
@@ -1,4 +1,9 @@
 export interface CopyCreatorOptions {
     from: string[];
     toDir: string;
+}
+
+export interface iconGenerateScript {
```

Comment:
interfaces with caps, lets try to keep it consistent everywhere

Reply:
sure @kbhutani0001, Will keep a check on this